cogs/servermanage/servermanage.py

```diff
@@ -292,7 +291,7 @@ def check(msg: discord.Message):
             self.logger.error("File does not exist %s", filepath)
 
         # Delete key from dictonary
-        async with getattr(self.config.guild(ctx.guild), imageType)() as images:
+        async with self.config.guild(ctx.guild).get_attr(imageType)() as images:
             del images[name]
         async with getattr(self.config.guild(ctx.guild), f"{imageType}Dates")() as dates:
```

Comment:
I believe this can be refactored as well.

Reply:
thanks for catching this